### PR TITLE
ci: prep for release

### PR DESCRIPTION
**Pull request by tomberek**
## Proposed Changes

- Trim lockfile
- better handling of release suffix

### CODE REVIEW

**Review by garbas — APPROVED** [review]

**Review by tomberek — COMMENTED** [review]
@aakropotkin

File: doc/release-notes/rl-0.9.0.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+ - Combined 'project environments' and 'managed environments' into a single 'environments' concept.
+ - Flox stores environment definition in a `.flox` folder in the current working directory by default.
+ - Flox enviornments support multiple architectures without having to create multiple environments. 
+ - The `--environment` or `-e` option for managed environments on many commands has been replaced with `-r` or `--remote` for FloxHub managed environments and `--dir` or `-d` to refer to local directory path environments.
+ - Environments are now delcared in a `manifest.toml` file embedded inside the `.flox` folder. `flox edit` will allow edits and validate changes made.
+ - The FloxHub service and its associated commands more broadly available to anyone with a valid GitHub account. Try the commands to sign up-- `flox push`, `flox pull`, `flox auth login`.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:10]:
```suggestion
 - The FloxHub service and its associated commands are more broadly available to anyone with a valid GitHub account. Try the commands to sign up-- `flox push`, `flox pull`, `flox auth login`.
```

File: doc/release-notes/rl-0.9.0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Release 0.9.0 (2024-02-15)
+
+This release contains **breaking changes in the Flox CLI** and FloxHub service. You are responsible for recreating environments created in 0.3.6 and prior versions. If you rely on a command that has been removed it is recommended to wait until support is added. We'd love to hear from you on our discourse forum if you have questions.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:3]:
```suggestion
This release contains **breaking changes in the Flox CLI** and FloxHub service. You are responsible for recreating environments created in 0.3.6 and prior versions. If you rely on a command that has been removed, we recommend that you wait to upgrade until support is added. We'd love to hear from you on our discourse forum if you have questions.
```

File: doc/release-notes/rl-0.9.0.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+ - Combined 'project environments' and 'managed environments' into a single 'environments' concept.
+ - Flox stores environment definition in a `.flox` folder in the current working directory by default.
+ - Flox enviornments support multiple architectures without having to create multiple environments. 
+ - The `--environment` or `-e` option for managed environments on many commands has been replaced with `-r` or `--remote` for FloxHub managed environments and `--dir` or `-d` to refer to local directory path environments.
+ - Environments are now delcared in a `manifest.toml` file embedded inside the `.flox` folder. `flox edit` will allow edits and validate changes made.
+ - The FloxHub service and its associated commands more broadly available to anyone with a valid GitHub account. Try the commands to sign up-- `flox push`, `flox pull`, `flox auth login`.
+ - Several deprecated sub commands were removed. Notably the `build` and `publish` command is no longer in Flox CLI. We have plans to return the build and publish features to Flox in the future. Consult the `--help` or `man flox` for a full list of supported commands.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:11]:
```suggestion
 - Several deprecated subcommands were removed. Notably the `build` and `publish` commands are no longer available in Flox CLI. We have plans to return the build and publish features to Flox in the future. Consult the `--help` or `man flox` for a full list of supported commands.
```

File: doc/release-notes/rl-0.9.0.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ - The Flox catalog that powers search and install is now tied to nixpkgs `23.11` stable branch. We have short term plans to expand the library of software in the Flox Catalog. `update` and `upgrade` will advance metadata index and software versions along the stable `23.11` branch.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:12]:
```suggestion
 - The Flox catalog that powers search and install is now tied to NixPkgs `23.11` stable branch. We have short term plans to expand the library of software in the Flox Catalog. `update` and `upgrade` will advance metadata index and software versions along the stable `23.11` branch.
```

**Review by rossturk — CHANGES_REQUESTED** [review]

File: doc/release-notes/rl-0.9.0.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+## Release 0.9.0 (2024-02-15)
+
+This release contains **breaking changes in the Flox CLI** and FloxHub service. You are responsible for recreating environments created in 0.3.6 and prior versions. If you rely on a command that has been removed, we recommend that you wait to upgrade until support is added. We'd love to hear from you on our discourse forum if you have questions.
+
+ - Combined 'project environments' and 'managed environments' into a single 'environments' concept.
+ - Flox stores environment definition in a `.flox` folder in the current working directory by default.
+ - Flox enviornments support multiple architectures without having to create multiple environments.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:7]:
```suggestion
 - Flox environments support multiple architectures without having to create multiple environments. 
```

File: doc/release-notes/rl-0.9.0.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+## Release 0.9.0 (2024-02-15)
+
+This release contains **breaking changes in the Flox CLI** and FloxHub service. You are responsible for recreating environments created in 0.3.6 and prior versions. If you rely on a command that has been removed, we recommend that you wait to upgrade until support is added. We'd love to hear from you on our discourse forum if you have questions.
+
+ - Combined 'project environments' and 'managed environments' into a single 'environments' concept.
+ - Flox stores environment definition in a `.flox` folder in the current working directory by default.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:6]:
```suggestion
 - Flox now stores environment definition in a `.flox` folder in the current working directory by default.
```

File: doc/release-notes/rl-0.9.0.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+ - Flox enviornments support multiple architectures without having to create multiple environments. 
+ - The `--environment` or `-e` option for managed environments on many commands has been replaced with `-r` or `--remote` for FloxHub managed environments and `--dir` or `-d` to refer to local directory path environments.
+ - Environments are now delcared in a `manifest.toml` file embedded inside the `.flox` folder. `flox edit` will allow edits and validate changes made.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:9]:
```suggestion
 - Environments are now declared in a `manifest.toml` file located inside the `.flox` folder. `flox edit` will allow edits and validate changes made.
```

File: doc/release-notes/rl-0.9.0.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+ - The FloxHub service and its associated commands are more broadly available to anyone with a valid GitHub account. Try the commands to sign up-- `flox push`, `flox pull`, `flox auth login`.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:10]:
```suggestion
 - The FloxHub service and its associated commands are now available to anyone with a valid GitHub account. These commands will direct you to sign up: `flox push`, `flox pull`, `flox auth login`.
```

File: doc/release-notes/rl-0.9.0.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+ - Several deprecated subcommands were removed. Notably the `build` and `publish` commands are no longer available in Flox CLI. We have plans to return the build and publish features to Flox in the future. Consult the `--help` or `man flox` for a full list of supported commands.
+ - The Flox catalog that powers search and install is now tied to NixPkgs `23.11` stable branch. We have short term plans to expand the library of software in the Flox Catalog. `update` and `upgrade` will advance metadata index and software versions along the stable `23.11` branch.
```
Comment (rossturk) [inline doc/release-notes/rl-0.9.0.md:12]:
```suggestion
 - The Flox catalog that powers search and install is now tied to the NixPkgs `23.11` stable branch. We have short term plans to expand the library of software in the Flox Catalog. Currently `update` and `upgrade` will advance metadata index and software versions along the stable `23.11` branch.
```

**Review by rossturk — CHANGES_REQUESTED** [review]

**Pull request merged**